dataline-server/src/main/java/io/dataline/server/handlers/SourceImplementationsHandler.java

```diff
@@ -65,7 +67,7 @@ public SourceImplementationRead createSourceImplementation(SourceImplementationC
     // validate configuration
     validateSourceImplementation(
         sourceImplementationCreate.getSourceSpecificationId(),
-        (String) sourceImplementationCreate.getConnectionConfiguration());
+        Jsons.jsonNode(sourceImplementationCreate.getConnectionConfiguration()));
```

Comment:
i'm assuming you didn't have any luck in getting the open api codegen to recognize `type: object` as `JsonNode`, right? or is that coming later?

Reply:
I actually did, let me remove this unecessary conversion